features/steps/thens/relations.py

```diff
@@ -189,108 +215,141 @@ def make_obj(s):
 
         accepted_values['applicable_entities'] = [s.split('/')[0] for s in make_obj(property_set_attr['applicable_entities'])]
 
-        accepted_values['applicable_type_values'] = property_set_attr.get('applicable_type_value', '').split(',')
+        # in the ifc4 data, predefined type restrictions are imposed as:
```

Comment:
I like this idea, but let's park it for a later version. The alternative is to make the CSV values of the 2x3 and 4+ versions more identical so that different code is no longer necessary.